docs/guides/stay-up-to-date/terraform/terraform-13/deployment-walkthrough/step-4-update-provider-sources.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: Update provider sources
+---
+
+# Step 4: update provider sources to the new format
+
+Terraform 13 introduced a new registry for managing provider releases. This makes it easier to publish and distribute
```

Comment:
```suggestion
Terraform 0.13 introduced a new registry for managing provider releases. This makes it easier to publish and distribute
```

Reply:
NOTE: I didn't accept this suggestion, because the hashes will be different so it needs to be regeneratd.